src/styles/styles.ts

```diff
@@ -3953,23 +3953,15 @@ const styles = (theme: ThemeColors) =>
             paddingBottom: 0,
         },
 
-        walletDangerSection: {
-            backgroundColor: theme.dangerSection,
-            color: theme.dangerSection,
-            borderRadius: variables.componentBorderRadiusCard,
-            width: 'auto',
-            marginHorizontal: 20,
-            marginBottom: 6,
-        },
-
-        walletDangerSectionTitle: {
-            fontSize: variables.fontSizeNormal,
-            fontFamily: fontFamily.EXP_NEUE_BOLD,
+        walletRedDotSectionTitle: {
+            color: theme.text,
             fontWeight: fontWeightBold,
+            fontSize: variables.fontSizeNormal,
             lineHeight: variables.lineHeightXLarge,
         },
 
-        walletDangerSectionText: {
+        walletRedDotSectionText: {
+            color: theme.darkSupportingText,
```

Comment:
We should use `color: theme.textSupporting` here.

More details:
https://github.com/Expensify/App/issues/61199